staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2683,8 +2683,19 @@ type InterfaceBindingPlugin struct {
 	// If namespace is not specified, VMI namespace is assumed.
 	// version: 1alphav1
 	NetworkAttachmentDefinition string `json:"networkAttachmentDefinition,omitempty"`
+	// DomainAttachment is a standard domain definition that kubevirt supports.
+	// Currently, only "Tap" is supported.
+	// The standard domain attachment can be used instead or in addition to the sidecar,
+	// to define the libvirt domain.
+	DomainAttachment DomainAttachmentType `json:"domainAttachment,omitempty"`
```

Comment:
Does it make sense to add `Type` to the field name as well?
As it is now a plain string, future domain attachment related params could use the same prefix with a specialized suffix.